spec/lib/email/sender_spec.rb

```diff
@@ -666,7 +666,7 @@ def stub_deliver_response(message)
 
           @secure_image_2 =
             UploadCreator.new(
-              file_from_fixtures("logo-dev.png", "images"),
+              file_from_fixtures("logo.png", "images"),
```

Comment:
Regardless of lightbox min height/width `logo-dev.png` would never have the `Content-ID` attached when it was in a lightbox div and used in email digest, however `logo.png` would even when in lightbox. I'm not sure if it's an issue with the test or fixture, or something else within secure uploads.